docs/source/AdministratorGuide/Systems/MonitoringSystem/index.rst

```diff
@@ -124,56 +134,32 @@ You can configure the MQ in the local dirac.cfg file where the agent is running:
 
 
-Enable PilotsHistory monitoring
-===============================
-In order to enable PilotsHistory monitoring you need to set the flag ``monitoringEnabled = True`` in Operations/Defaults.
+Monitoring of DIRAC Agents and Services
+=======================================
 
+When enabled, this will report the activity of DIRAC agents and services, including parameters such as CPU and Memory usage, but also cycle duration of
+agents, or response time, queries and threads of the services.
 
-Enable Monitoring of DIRAC Agents and Services
-==============================================
 
-You have to set ``EnableActivityMonitoring=True`` in the CS.
-It can be done globally, the ``Operations`` section, or per single component.
+RMS Monitoring
+==============
 
+This type is used to monitor behaviour pattern of requests executed by RequestManagementSystem inside DataManagementSystem/Agent/RequestOperations.
 
-Enable RMS Monitoring
-=====================
+PilotSubmission Monitoring
+==========================
 
-In order to enable RMSMonitoring we need to set value of ``EnableRMSMonitoring`` flag to yes/true in the CS::
+This monitoring type reports statistics of the pilot submissions done by the SiteDirector, including parameters such as the total number of submissions and the succeded ones.
 
+Data Operation Monitoring
+=========================
 
-   Systems
-   {
-     RequestManagement
-     {
-       <instance>
-       {
-         Agents
-         {
-           RequestExecutingAgent
-           {
-             ...
-             EnableRMSMonitoring = True
-           }
-         }
-       }
-     }
-   }
-
-Enable Pilot Submission Monitoring
-==================================
-
-In order to enable the monitoring of the pilot submission so that they will be sent to ES backend (by default they are sent to Accounting), you need to set
-``sendPilotSubmissionMonitoring = True`` for this option in WorkloadManagement/SiteDirector.
-
-Enable Data Operation Monitoring
-==================================
-
-To enable the monitoring of data operation to have it sent to ES backend, you need to add ``Monitoring`` to the ``MonitoringBackends`` option in Operations/DataManagement.
-
+This monitoring enables the reporting of information about the data operation such as the cumulative transfer size or the number of succeded and failed transfers.
 
 
 Accessing the Monitoring information
 =====================================
 
-After you installed and configured the Monitoring system, you can use the Monitoring web application.
+After you installed and configured the Monitoring system, you can use the Monitoring web application for the types WMSHistory, PilotSubmission and DataOperation.
+
+However, every type can directly be monitored in the Kibana dashboards of the ElasticSearch instance. These can be found and imported from DIRAC.
```

Comment:
Can you expand on this?

Reply:
Yes, I will also link to the dashboards that are in the making in https://github.com/DIRACGrid/DIRAC/pull/6021